### PR TITLE
{bio}[GCC/12.3.0,GCCcore/12.3.0,foss/2023a] Circlator v1.5.5, MUMmer v4.0.0rc1, Fastaq v3.17.0

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/b6ea54dc5037038095e1ad6fc2899264 for a full test report.

File: easybuild/easyconfigs/c/Circlator/Circlator-1.5.5-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pymummer', '0.11.0', {
+        'checksums': ['199b8391348ff83760e9f63fdcee6208f8aa29da6506ee1654f1933e60665259'],
+    }),
+    (name, version, {
+        'source_urls': [
+            'https://files.pythonhosted.org/packages/7c/bf/f0fd534a733b7d98fc9853ca1aa735984cec1e5fc4670b632e76973f8add/'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Circlator/Circlator-1.5.5-foss-2023a.eb:31]: You can avoid having to specify `source_urls` at all by using lowercase name as extension:

```python
('circulator', version, {
```

`PythonBundle` by default knows that extensions should be downloaded via PyPI, you don't need to specify that yourself

File: easybuild/easyconfigs/c/Circlator/Circlator-1.5.5-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        ],
+        'sources': ['circlator-1.5.5.tar.gz'],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Circlator/Circlator-1.5.5-foss-2023a.eb:33]: Please don't hardcode the version here, use:

```python
        'sources': ['circlator-%(version)s.tar.gz'],

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download --sourcepath /tmp/$USER/19847"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/3fac281be7f5f137d8b7cb1d9953631b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**